zaza/charm_lifecycle/func_test_runner.py

```diff
@@ -13,9 +13,9 @@
 import zaza.charm_lifecycle.test as test
 
 
-def generate_model_name(charm_name, bundle_name):
+def generate_model_name():
     timestamp = datetime.datetime.now().strftime('%Y%m%d%H%M%S')
-    return 'zaza-{}{}{}'.format(charm_name, bundle_name, timestamp)
+    return 'zaza-{}'.format(timestamp)
```

Comment:
I like reducing this to timestamp, but I would like to consider more precision on the timestamp, given that we often en-queue test matrices, which can cause multiple deploys to start at nearly the same time. I suspect that we would be fine simply because we don't share controllers between the various test runners on UOSCI but I wouldn't want to implicitly depend on that behaviour